app/static/js/game_releases.js

```diff
@@ -0,0 +1,17 @@
+get_games_btn = document.getElementById("get-games");
+get_games_btn.addEventListener("click", function (e) {
+  let formData = new FormData();
+  formData.append("from-date", document.getElementById("from-date").value);
+  formData.append("to-date", document.getElementById("to-date").value);
+  let clickedElem = e.target.id;
+  fetch("/game-releases/get_releases_by_dates", {
+    method: "post",
+    body: formData,
+  })
+    .then(function (response) {
+      return response.text();
+    })
+    .then(function (body) {
+      document.querySelector("#content-div").innerHTML = body;
```

Comment:
innerText maybe?

Reply:
![image](https://user-images.githubusercontent.com/52039539/109323788-834b3300-785c-11eb-83d6-928700dd9da8.png)
I'm not sure it would work here, wouldn't I be lacking formatting?